### PR TITLE
Move EIP-712 to Final

### PR DESCRIPTION
**Pull request by frangio**
There have been no comments since [Last Call was announced](https://ethereum-magicians.org/t/eip-712-eth-signtypeddata-as-a-standard-for-machine-verifiable-and-human-readable-typed-data-signing/397/35?u=frangio).

### CODE REVIEW

**Review by Pandapip1 — COMMENTED** [review]
The formatting is a bit wonky, but otherwise LGTM. Since this is a pretty significant EIP, I'm going to hold off from flat-out approving it and will wait for someone else to also signal approval.

**Comment by frangio** [comment]
@Pandapip1 Do you mean the images? I agree but I'm not sure what's the right way to resize them that will work on the final rendered site. Does anyone else know?

**Comment by Pandapip1** [comment]
Not the images; the spacing in between headers is inconsistent, as well as a few other things. I don't think they result in any issues.

**Review by SamWilsn — APPROVED** [review]

**Comment by Pandapip1** [comment]
Please close and re-open this PR to merge it.

**Comment by frangio** [comment]
@Pandapip1 Should I do that before or after author approval? (Note: I'm not author)

**Comment by Pandapip1** [comment]
It has been approved.

**Review by Pandapip1 — APPROVED** [review]

**Comment by Pandapip1** [comment]
CC @recmo, @LogvinovLeon, @expede

**Comment by wschwab** [comment]
I only noticed that 712 made Last Call now - I had originally written a resolution to the checklist that originally was put in the Security Considerations, which I think is removed in this version of 712. Would I be able to put it back in?

Reference: https://github.com/ethereum/EIPs/pull/4383

**Review by LogvinovLeon — APPROVED** [review]

**Comment by frangio** [comment]
@wschwab We discussed that section in https://github.com/ethereum/EIPs/pull/5271#discussion_r923724798 with the editors and agreed that it felt out of place in the EIP as it was more of an implementation log.

**Pull request merged**
